cpp/tests/utilities/cudf_test_utils.cuh

```diff
@@ -36,6 +38,38 @@
 using gdf_col_pointer = typename std::unique_ptr<gdf_column, 
                                                  std::function<void(gdf_column*)>>;
 
+
+// Prints a vector and valids
+template <typename T>
+void print_vector_and_valid(T * v,
```

Comment:
This looks to be a better replacement for `print_typed_column`. Please remove `print_typed_column` and update code that uses it.

Reply:
Function removed.